maccabipediabot/common.py

```diff
@@ -16,16 +16,23 @@ def create_maccabipedia_shirt_number_category_html_text(shirt_number):
 
 
 def get_song_lyrics(song_name):
+    """
+    scrapes the given song from it's page at maccabipedia, parsing the HTML to string and return the lyrics
+    :param song_name: Tha name of the song
+    :type song_name: str
+    :return: link to the given song at maccabipedia & the lyrics of the given song
+    :rtype: str
+    """
     url = _MACCABIPEDIA_LINK + '/שיר:' + song_name
```

Comment:
I think we can write a log for each time we crawl maccabipedia (with the link, will help to debug).

Reply:
let's do it :) 👍